aws/resource_aws_backup_plan.go

```diff
@@ -69,6 +69,35 @@ func resourceAwsBackupPlan() *schema.Resource {
 								},
 							},
 						},
+						"copy_action": {
+							Type:     schema.TypeList,
```

Comment:
If this attribute supports more than 1, we should add a covering acceptance test configuration to determine if the API orders results (or requires `TypeSet`), otherwise, we should ensure this has `MaxItems: 1` as well. 👍